yarn.lock

```diff
@@ -2,9 +2,9 @@
 # yarn lockfile v1
 
 
-"@babel/cli@7.0.0-rc.2":
-  version "7.0.0-rc.2"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.0.0-rc.2.tgz#0c64762f27ccd3fa255111570b6c404f81181c85"
+"@babel/cli@^7.0.0":
```

Comment:
I'm not sure why Babel changed here? make-dir doesn't depends on Babel. Maybe we forgot to update the lock file after our own Babel bump.